playground/main.ts

```diff
@@ -159,4 +165,12 @@ if (typeof URL !== 'undefined') {
     render = false;
   }
 }
-render && renderPlaygroundWidget(window.additionalOptions ?? {});
+
+// set up msw
+getWorker()
```

Comment:
Quick question on this, I recall (a very long time ago 😅 ) there was an odd issue when we used both MSW with the Dyson mock server, where the same request would be triggered multiple times. Is this no longer an issue?

Reply:
I guess not, I don't see that happening when running the testcafe suits